onnxruntime/test/providers/cpu/nn/qlinearconv_op_test.cc

```diff
@@ -409,27 +432,60 @@ class QLinearConvOpTester {
           float weight_scale = W_.scale_[(W_.scale_.size() == 1) ? 0 : channel_index];
           float requantize_scale = (X_.scale_[0] * weight_scale) / output_scale_;
 
-          for (int64_t oh = 0; oh < output_h; oh++) {
-            for (int64_t ow = 0; ow < output_w; ow++) {
+          if (kernel_rank == 2) {
+            for (int64_t oh = 0; oh < output_h; oh++) {
```

Comment:
We don't need to special case the kernel_rank=2 here for test code: just use the ND case for everything.

Reply:
Agree. This is just test, no strong perf requirement.